pyServer/GuestFishWrapper.py

```diff
@@ -79,6 +80,13 @@ def WriteToResultFile(self, operationOutFile, data):
         operationOutFile.write('\r\n') 
         operationOutFile.flush()
 
+    def WriteToScanFileListFile(self, scanFileListFile, data):
```

Comment:
Can we consider renaming the parameter 'data' to something like 'filename' to be more descriptive?

Reply:
Changed to filename. At the time, I wrote this utility method first. I was thinking to be generic to be able to write any data. In the end, didn't go back to change it. good suggestion